julia/oxygen/Project.toml

```diff
@@ -0,0 +1,3 @@
+[deps]
+Oxygen = "df9a0d86-3283-4920-82dc-4555fc0d1d8b"
+HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
```

Comment:
Maybe this should be dropped. This is already defined in Oxygen's project.toml

Reply:
@AbrJA

Reply:
I tried to avoid it but I got an error, let me check another way, I will test it locally to be sure

Reply:
I was reviewing the implementation of @ndortega and he also include this into the [Project.toml](https://github.com/TechEmpower/FrameworkBenchmarks/pull/8789/files#diff-ad1f5944da489952393483f4aee90304851db60a10fa8dfa9263bfde68677495) Maybe the best option is to keep it

Reply:
seems redundant for me, but let @ndortega gives his opinion

Reply:
@ndortega What do you think?